packages/integrations/vercel/src/edge/adapter.ts

```diff
@@ -66,6 +73,9 @@ export default function vercelEdge(): AstroIntegration {
 						target: 'webworker',
 						noExternal: true,
 					};
+
+					vite.build ||= {};
+					vite.build.minify = true;
```

Comment:
I believe Vite minifies by default, and the other edge integrations doesn't set this too. Maybe this can be removed?

Reply:
Hmm. I don't know if it's because I'm in a development environment, but I need to set this to true for it to work

Reply:
Also, Vercel Edge Functions are limited to 1 MB in size. This change is necessary

Reply:
Ah maybe this line is causing it:

https://github.com/withastro/astro/blob/49a8d18b4993d899a05ee8230fdd012fb633533f/packages/astro/src/core/build/static-build.ts#L140

In that case forcing `minify: true` sounds good then. We should probably do the same for the other integrations.